libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32H755xx.py

```diff
@@ -0,0 +1,1304 @@
+#!/usr/bin/env python
+'''
+these tables are generated from the STM32 datasheets for the STM32H755
+'''
+
+# additional build information for ChibiOS
+build = {
+    "CHIBIOS_STARTUP_MK"  : "os/common/startup/ARMCMx/compilers/GCC/mk/startup_stm32h7xx.mk",
+    "CHIBIOS_PLATFORM_MK" : "os/hal/ports/STM32/STM32H7xx/platform.mk"
+    }
+
+# MCU parameters
+mcu = {
+    # location of MCU serial number
+    'UDID_START' : 0x1FF1E800,
+
+    # DMA peripheral capabilities:
+    # - can't use ITCM or DTCM for any DMA
+    # - SPI1 to SPI5 can use AXI SRAM, SRAM1 to SRAM3 and SRAM4 for DMA
+    # - SPI6, I2C4 and ADC3 can use SRAM4 on BDMA
+    # - UARTS can use AXI SRAM, SRAM1 to SRAM3 and SRAM4 for DMA
+    # - I2C1, I2C2 and I2C3 can use AXI SRAM, SRAM1 to SRAM3 and SRAM4 with DMA
+    # - timers can use AXI SRAM, SRAM1 to SRAM3 and SRAM4 with DMA
+    # - ADC12 can use AXI SRAM, SRAM1 to SRAM3 and SRAM4
+    # - SDMMC can use AXI SRAM, SRAM1 to SRAM3 with IDMA (cannot use SRAM4)
+
+    # ram map, as list of (address, size-kb, flags)
+    # flags of 1 means DMA-capable (DMA and BDMA)
+    # flags of 2 means faster memory for CPU intensive work
+    # flags of 4 means memory can be used for SDMMC DMA
+    'RAM_MAP' : [
+        (0x30000000, 256, 0), # SRAM1, SRAM2
+        (0x20000000, 128, 2), # DTCM, tightly coupled, no DMA, fast
+        (0x24000000, 512, 4), # AXI SRAM. Use this for SDMMC IDMA ops
+        (0x00000400,  63, 2), # ITCM (first 1k removed, to keep address 0 unused)
+        (0x30040000,  32, 0), # SRAM3.
+        (0x38000000,  64, 1), # SRAM4.
+    ],
+    # alternative RAM_MAP needed for px4 bootloader compatibility
```

Comment:
likely don't need ALT_RAM_MAP

Reply:
@tridge 

We considered stm32h755 and stm32h757 to be the same architecture.
I am using a copy of stm32h757.

does it incorrect?

Reply:
I checked it works correctly.
so I deleted it.